rdrf/rdrf/templates/registration/login_assistance_account_not_locked_email.html

```diff
@@ -0,0 +1,23 @@
+{% load i18n %}{% autoescape off %}
+Hello  Mr/Ms {{user.last_name}},
+
+{% blocktrans %}You're receiving this email because you were having trouble signing in and requested help.{% endblocktrans %}
+
+{% blocktrans %}Your account is NOT locked. You should be able to log in if you're providing the correct username and password.{% endblocktrans %}
```

Comment:
Minor comment - I don't think we need to tell the user the account is not locked.